notebooks/203-meter-reader/203-meter-reader-with-paddle-ocr.ipynb

```diff
@@ -0,0 +1,925 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #46.    def post_processing(struct_result):
</pre><p>Why shall we create a new <code>post_processing</code> function here, could we combine it with the previous one and filter the readings by configurations ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1030/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>